gui/src/main.rs

```diff
@@ -241,11 +252,11 @@ fn run_launcher(
 
     win.on_start_vmm({
         let win = win.as_weak();
-        let start = start.clone();
+        let debug_addr = debug_addr.clone();
 
         move || {
             win.unwrap().hide().unwrap();
-            start.set(true);
+            debug_addr.set(Some(None));
```

Comment:
Does this mean we always start the kernel in debug when pressing Start button?

Reply:
I think I understand it now. Better to create a dedicated enum for this since nested `Option` here is really confusing.